internal/api/v1/stage.go

```diff
@@ -206,6 +194,11 @@ func (hc ApplicationsController) Stage(w http.ResponseWriter, r *http.Request) A
 		return InternalError(err, "failed to ensure a PersistenVolumeClaim for the application source and cache")
 	}
 
+	cs, err := tekton.NewForConfig(cluster.RestConfig)
+	if err != nil {
+		return InternalError(err, "failed to get access to a tekton client")
+	}
+	client := cs.TektonV1beta1().PipelineRuns(deployments.TektonStagingNamespace)
```

Comment:
Use this when merged? https://github.com/epinio/epinio/pull/862/files#diff-e02c24d0a09d47ddaca26d2c3540697e3dae4ce4f296b3f7448c6f34dc6dfc12R198